components/ui/MovieList.tsx

```diff
@@ -0,0 +1,29 @@
+// components/MovieList.tsx
+import React from 'react';
+
+interface Movie {
+  Title: string;
+  Year: string;
+  imdbID: string;
+  Poster: string;
+}
+
+interface MovieListProps {
+  movies: Movie[];
+}
+
+const MovieList: React.FC<MovieListProps> = ({ movies }) => {
```

Comment:
**suggestion**: this is a personal taste, but perhaps you could create your components like this:

```tsx
function MovieList({ movies }: MovieListProps): {
  // rest of your code here
}
```